tests/postgres_tests/test_ranges.py

```diff
@@ -1104,3 +1106,15 @@ def test_range_widget_render_tuple_value(self):
             '<input type="text" name="datetimerange_0" value="2022-04-22 10:24:00">'
             '<input type="text" name="datetimerange_1" value="2022-05-12 09:25:00">',
         )
+
+
+@modify_settings(INSTALLED_APPS={"append": "django.contrib.postgres"})
+class TestCheckConstraintsOnRange(PostgreSQLTestCase):
+    def test_check_constraint_nullable_value_range(self):
+        instance = NullableIntegerRangeCheckConstraintModel()
+        with self.assertRaises(exceptions.ValidationError) as cm:
+            instance.validate_constraints()
```

Comment:
We could also assert that `DEBUG` logger doesn't contain a message with `operator does not exist`.

Reply:
I'm not entirely sure what the best approach to do this is. If had the liberty of writing code for only 3.10 we could use [`assertNoLogs()`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertNoLogs)

Reply:
Django provides `assertNoLogs()` fallback for Python < 3.10 (7ca7f4495ba746279b734695a8dd137bf7ee0bab).

Reply:
@felixxm I ended up using `assertNoLogs("django.db.models", "DEBUG")` although this may break in the future if something happened to start logging debug level logs during validation. It doesn't look possible to use `assertNoLogs()` against specific wording – unless we mock logging.  One thing we could do is change the level to WARNING?

Reply:
Finally, I decided that it's not worth testing `assertNoLogs()`.